server/region_syncer/server.go

```diff
@@ -95,7 +96,7 @@ func NewRegionSyncer(s Server) *RegionSyncer {
 		tlsConfig: s.GetTLSConfig(),
 	}
 	syncer.mu.streams = make(map[string]ServerStream)
-	syncer.mu.closed = make(chan struct{})
+	syncer.mu.clientCtx, syncer.mu.clientCancel = context.WithCancel(context.Background())
```

Comment:
Just a question, why not create a sub-context from the server rather than using a `context.Background()`?